tests/listobjects/listobjects.go

```diff
@@ -108,19 +109,27 @@ func runTests(t *testing.T, schemaVersion string, client ListObjectsClientInterf
 					require.NoError(t, err)
 				}
 
-				for _, assertion := range stage.Assertions {
+				for _, assertion := range stage.ListObjectAssertions {
 					// assert 1: on regular list objects endpoint
+					t.Logf("ListObject request: %s. Contextual tuples: %s", assertion.Request, assertion.ContextualTuples)
```

Comment:
See https://github.com/openfga/openfga/pull/601#discussion_r1125309481 - we need this.  Otherwise, it becomes very difficult to tell if there are more than 1 list obj assertion

Reply:
It only appears if you run `go test -v` or if the test fails

Reply:
If you need to provide more context in the logged failure message, then that's the good use for the `msg` field in the `require` assertion. I think that's a better approach than helper logging in the tests. For example, in this case it'd be more advantageous to do:

```
extraErrorMsg := "Expectation failed. ListObjects request (%s), ContextualTuples (%s)"

...
require.Equal(t, ...., extraErrorMsg)
```

This way you literally only see the message if an expectation was  not met, and in that case you'll have clear context where it came from.

Reply:
swap to use extraErrorMsg as suggested above